sdks/python/apache_beam/transforms/util_test.py

```diff
@@ -902,6 +904,19 @@ def test_in_global_window(self):
                       GroupIntoBatchesTest.BATCH_SIZE))
           ]))
 
+  def test_in_global_window_with_text_file(self):
+    with self.assertRaises(NotImplementedError):
```

Comment:
note this test just confirms the code path with output_handler works but the feature is not implemented for DirectRunner